packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2729,12 +2729,20 @@ class InputDecoration {
   /// [errorText], and [counterText].
   ///
   /// By default the `contentPadding` reflects [isDense] and the type of the
-  /// [border]. If the [border] is [OutlineInputBorder] then `contentPadding` is
-  /// `EdgeInsets.LTRB(12, 24, 12, 16)`. If the [border] is [UnderlineInputBorder]
-  /// or [InputBorder.none] then `contentPadding` is `EdgeInsets.LTRB(0, 12, 0, 12)`.
-  /// If the [border] is [UnderlineInputBorder] or [InputBorder.none] and if
-  /// [filled] is true then `contentPadding` is `EdgeInsets.LTRB(12, 12, 12, 12)`.
+  /// [border].
+  ///
   /// If [isCollapsed] is true then `contentPadding` is [EdgeInsets.zero].
+  ///
+  /// If [InputBorder.isOutline] is false and [filled] is true then
```

Comment:
We might want to still link this into [border] somehow so people know the parameter to set. Maybe:
```suggestion
  /// If [border]'s [InputBorder.isOutline] property is false and [filled] is true then
```